flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -391,6 +397,154 @@ static void msp_send_boxids(struct msp_bridge *m) {
 	msp_send(m, MSP_BOXIDS, boxes, len);
 }
 
+static const char alarm_names[][8] = {
```

Comment:
Yes, you always need to defined the size of the second and subsequent dimensions.